apps/api/src/friendships/friendships.controller.ts

```diff
@@ -0,0 +1,32 @@
+import { Controller, Post, Delete, Body, Get, Query, UseGuards } from "@nestjs/common";
+import { FriendshipsService } from "./friendships.service";
+// import { JwtAuthGuard } from '../auth/jwt-auth.guard';
+import { User } from "types";
+
+//@UseGuards(JwtAuthGuard)
+@Controller('friendships')
+export class FriendshipsController {
+    constructor(
+        private readonly fiendshipsService : FriendshipsService
+    ) {}
+
+    @Post()
+    async add (@Body() body : {initiator_id: string, target_id: string}) : Promise<number> {
+        return this.fiendshipsService.add(body.initiator_id, body.target_id);
+    }
+
+    @Get()
```

Comment:
Same, get current user id from decorator, get rid of query, can be changed after merge